src/modules/Bootloader.rb

```diff
@@ -418,7 +418,6 @@ def modify_kernel_params(*args)
 
           case value
           when false then next # already done
-          when true then param.add_parameter(key, value)
           when Array
             value.each { |val| param.add_parameter(key, val) }
```

Comment:
Is is `true.each { ...` now? If so, you know,  `undefined method each' for true:TrueClass)`

Reply:
no, true is not Array...it goes to else section which is `param.add_parameter(key, value)`

Reply:
Right! I overlooked else and mixed branches at the time of review :)